presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -982,7 +986,7 @@ export class QueryDetail extends React.Component {
 
     renderPreparedQuery() {
         const query = this.state.query;
-        if (query.preparedQuery === null) {
+        if (!query.hasOwnProperty('preparedQuery') || query.preparedQuery === null) {
```

Comment:
why not 

```
if (query.preparedQuery === undefined || query.preparedQuery === null)
```

?

Reply:
or 
```
if (!query.preparedQuery)
```

Reply:
Habit + query has no shape defined in React State